packages/gatsby-plugin-theme-ui/src/provider.js

```diff
@@ -3,7 +3,7 @@ import { jsx, ThemeProvider, ColorMode } from 'theme-ui'
 import theme from './index'
 import components from './components'
 
-export const wrapRootElement = ({ element }, opts) =>
+export const wrapRootElement = ({ element }, _opts) =>
```

Comment:
I think this was probably a remnant from the other plugin approach or the color mode stuff

```suggestion
export const wrapRootElement = ({ element }) =>
```